systems/plants/test/evaluateNewSecondDerivativeMethod.m

```diff
@@ -0,0 +1,75 @@
+function evaluateNewSecondDerivativeMethod()
+%NOTEST
```

Comment:
i think it's fine to commit since it's in test.  we also sometimes have dev subdirs.